### DISCUSSION TITLE
Step by step guide to install pybricks firmware to Spike/Inventor-Hub?

### DISCUSSION THREAD

**Discussion by Lebostein** [pybricks/support#528] [Q&A]
Where I can find a guide to install the pybricks firmware on a Spike/Inventor-Hub?

**Comment by laurensvalk** [comment]
It's still in its alpha state, so we haven't put the instructions on our main page. If that doesn't bother you, you can follow these steps: https://github.com/pybricks/support/issues/167

**Reply by Lebostein** [reply]
Thanks!

PS: Alpha or beta? 

> Pybricks for LEGO Education SPIKE Prime and LEGO MINDSTORMS Robot Inventor is currently still in **beta**.

**Reply by laurensvalk** [reply]
I guess we could be a bit more consistent with naming. Unfinished either way. 😄
